src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -33,12 +34,18 @@ internal TagHelperOutput(string tagName)
         /// <param name="tagName">The HTML element's tag name.</param>
         /// <param name="attributes">The HTML attributes.</param>
         public TagHelperOutput(string tagName, [NotNull] IDictionary<string, string> attributes)
+            : this(tagName, attributes, HtmlEncoder.Default)
```

Comment:
Sure, but in the majority case users will never provide an `IHtmlEncoder` to begin with so why enforce providing it in `TagHelperOutput`s construction? If a `TagHelper` author is serious about the `TagHelper`s they create they'll be explicit as to which encoder they want used.